acceptance-tests/src/test-support/java/tech/pegasys/artemis/test/acceptance/dsl/ArtemisNode.java

```diff
@@ -147,27 +165,37 @@ public void stop() {
       return;
     }
     LOG.debug("Shutting down");
-    if (!configFile.delete() && configFile.exists()) {
-      throw new RuntimeException("Failed to delete config file: " + configFile);
-    }
+    configFiles.forEach(
+        configFile -> {
+          if (!configFile.delete() && configFile.exists()) {
+            throw new RuntimeException("Failed to delete config file: " + configFile);
+          }
+        });
     container.stop();
   }
 
   public static class Config {
 
+    private final PrivKey privateKey = KeyKt.generateKeyPair(KEY_TYPE.SECP256K1).component1();
+    private final PeerId peerId = PeerId.fromPubKey(privateKey.publicKey());
```

Comment:
Might be nice to have a have some static `NodeId` methods for generating ids.  So we don't need to pull in `libp2p` dependencies.

Reply:
I'm generally trying to avoid depending on production code from the acceptance tests - they tend to get quickly intertwined and then bugs in one are reproduced in the tests rather than causing failures.  So while I'd prefer to use something else entirely to be able to generate keys I'd rather depend on jvmlibp2p than the Artemis code we're trying to test.